src/Controller/SentencesController.php

```diff
@@ -195,11 +195,12 @@ public function show($id = null)
             $tagsArray = $this->Sentences->getAllTagsOnSentence($id);
             $this->loadModel('SentencesSentencesLists');
             $listsArray = $this->SentencesSentencesLists->getListsForSentence($id);
```

Comment:
`$listsArray` is never used so that line is unnecessary.

But since your `getOwnAndPublicListsForSentence` is just a modified copy of `getListsForSentence` and that function was only used for getting the lists to show on the sentence page and so would be obsolete after merging the PR, I think you can simply include your changes in the old version.